modules/contact-form/grunion-contact-form.php

```diff
@@ -2816,7 +2825,15 @@ function process_submission() {
 			return self::success_message( $post_id, $this );
 		}
 
-		$redirect = wp_get_referer();
+		$redirect = '';
+		if ( 'redirect' === $this->get_attribute( 'customThankyou' ) ) {
+			$redirect = $this->get_attribute( 'customThankyouRedirect' );
```

Comment:
Should `customThankyouRedirect` be sanitized as an URL?

Reply:
It absolutely should be, good catch! 🙂